src/System.IO/src/project.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "Microsoft.DotNet.CoreCLR": "1.0.0-prerelease"
+  }
```

Comment:
@mellinoe @FiveTimesTheFun does having this reference mean we automatically reference mscorlib? I was kind of still expecting to see the mscorlib reference somewhere in the project file to make it clear.